browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -2841,9 +2841,13 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		var backupGridSpan = control.style.gridColumn;
 
 		control.replaceWith(temporaryParent.firstChild)
+		let onclick = control.onclick;
 
 		var newControl = container.querySelector('[id=\'' + elementId + '\']');
 		if (newControl) {
+			if (onclick) {
```

Comment:
This shouldn't be needed, something is wrong when calling "update" I think. To check.

Reply:
the conntrol is replaced so we need to move the handler over. Without that the clicked handler is gone.